tensorflow/compiler/mlir/tools/kernel_gen/kernel_creator.cc

```diff
@@ -198,14 +198,15 @@ Status LowerTFToJITInvocation(mlir::ModuleOp module,
                               llvm::ArrayRef<int64_t> tile_sizes,
                               llvm::ArrayRef<int64_t> unroll_factors,
                               int64_t max_supported_rank, bool enable_ftz,
-                              bool cpu_codegen) {
+                              bool index_64bit, bool cpu_codegen,
+                              bool jit_i64_indexed_for_large_tensors) {
   mlir::PassManager pm(module.getContext());
   applyTensorflowAndCLOptions(pm);
 
   pm.addNestedPass<mlir::FuncOp>(
       mlir::kernel_gen::transforms::CreateTFToJITInvocationPass(
```

Comment:
`CreateTFToJITInvocationPass` is missing one argument here: `index_64bit`
The default parameters are dangerous, but it's not easy to get away w/o them.

Reply:
Modified.